sklearn/model_selection/_search.py

```diff
@@ -203,6 +203,8 @@ class ParameterSampler:
 
     n_iter : integer
         Number of parameter settings that are produced.
+        ..versionchanged:: 0.20
```

Comment:
```suggestion

        ..versionchanged:: 0.20
```
@borovikova sorry... those sphinx things make me crazy...
If you would still trust me, you can try this one... Thanks for your patience!

Reply:
@cmarmo Thank you for your help!
There was one extra space character in front of `..versionchanged:: 0.20` that you removed, right? I'm just asking because it's hard to see with a naked eye :)

Reply:
This time I have added a blank line before `..versionchanged::0.20`. My guess is that without an additional line sphinx still thinks you are in the same paragraph and can't accept the indentation the line after the versionchanged directive ... but you know... it's a guess... :)

Reply:
I would add that blank line everywhere (even if not strictly necessary) for readability.